STARTERKIT/basic/kaizen.theme

```diff
@@ -1,11 +1,26 @@
 <?php
 
+/**
+ * Returns path to svg sprite.
+ */
+function kaizen_get_svg_sprite_path() {
+  return base_path() . \Drupal::theme()->getActiveTheme()->getPath() . '/dist/svg/sprite.svg';
+}
+
+/**
+ * hook_page_attachments().
+ *
+ * Prepares svg_sprite_path variable to get full path to svg sprite from js.
+ */
+function kaizen_page_attachments_alter(array &$attachments) {
+  $attachments['#attached']['drupalSettings']['svg_sprite_path'] = test_get_svg_sprite_path();
```

Comment:
the function name needs work `test_get_svg_sprite_path()` is no-go

Reply:
I mean it should be prefixed with theme name, a-la `_kaizen_get_svg_sprite_path`

Reply:
oh, forgot to rename, true